_search-plugins/search-pipelines/index.md

```diff
@@ -0,0 +1,387 @@
+---
+layout: default
+title: Search pipelines
+nav_order: 100
+has_children: true
+has_toc: false
+---
+
+# Search pipelines
+
+Search pipelines is an experimental feature. Therefore, we do not recommend the use of search pipelines in a production environment. For updates on the progress of search pipelines, or if you want to leave feedback that could help improve the feature, see the associated [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/6278).    
```

Comment:
```suggestion
Search pipelines is an experimental feature and therefore is not recommended for use in a production environment. For updates on the progress of search pipelines, or if you want to leave feedback that could help improve the feature, see the associated [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/6278).    
```

Reply:
I've used this wording before but let's ask @natebower's opinion if it should be changed.

Reply:
We reopened the RFC for this feature: https://forum.opensearch.org/t/rfc-search-pipelines/12099 to use instead of the GH issue.